src/controller/python/chip-device-ctrl.py

```diff
@@ -299,6 +328,8 @@ def do_setlogoutput(self, line):
         Set the level of Chip logging output.
         """
 
+        # warnings.warn("This method is being deprecated. Please use the DeviceController.SetLogFilter method directly in the REPL", DeprecationWarning)
```

Comment:
Do we want commented out code for future deprecation?
Consider a helper like `def warnDeprecated(self, alternative=None)` to have shorter lines.

Reply:
I'll need to add a bunch more in a future PR, so will re-factor then.